mirdata/validate.py

```diff
@@ -76,12 +76,12 @@ def validate_files(file_dict, data_home, verbose):
     missing = {}
     invalid = {}
     for file_id, file in tqdm.tqdm(file_dict.items(), disable=not verbose):
-        # multitrack case
```

Comment:
@rabitt, this gave me an error in validating the index we agreed upon so I had to move it one level down. 'mozart' being a key in the multitrack which has the subkeys 'tracks' (and maybe 'audio-mix').

Reply:
Nice. maybe this function deserves a refactor in the future. It's getting a little nest-y